runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/Index.stg

```diff
@@ -13,5 +13,5 @@ TestFolders ::= [
 	"SemPredEvalLexer": [],
 	"SemPredEvalParser": [],
 	"Sets": [],
-	"Visitors": []
+	"Visitors": "JavaScript/Node"
```

Comment:
❗️ I prefer to not add target-specific filtering to the test suite templates themselves. You can already exclude specific tests in the target-specific templates using the `IgnoredTests` map. If you want to modify the behavior so it omits the test altogether (as opposed to just adding the `@Ignore` annotation), that would be preferable to a third exclusion mechanism - all of which remain undocumented AFAIK.